projects/packages/publicize/src/class-share-limits.php

```diff
@@ -96,22 +96,7 @@ public function get_current_url() {
 	 */
 	public function render_classic_editor_notice() {
 		$current_url = $this->get_current_url();
-		$notice      = sprintf(
-			/* translators: %1$d: number of shares remaining, %2$s: link to upgrade the plan. */
-			_n(
-				'You currently have %1$d share remaining. <a href="%2$s" target="_blank">Upgrade</a> to get more.',
-				'You currently have %1$d shares remaining. <a href="%2$s" target="_blank">Upgrade</a> to get more.',
-				$this->shares_remaining,
-				'jetpack-publicize-pkg'
-			),
-			$this->shares_remaining,
-			Redirect::get_url(
-				'jetpack-social-basic-plan-classic-editor',
-				array(
-					'query' => 'redirect_to=' . rawurlencode( $current_url ),
-				)
-			)
-		);
```

Comment:
Should we use the feature flag here before nuking this?

Reply:
Do we have a feature flag for the plan changes already? 🤔  but yeah, good point, if we use the flag we do not have to delay merging it

Reply:
How about `$publicize->has_paid_features()`?

Reply:
Yeah that would work if they have the Advanced plan, but I think then free users would still see it, as it would be false for them, but after we retire the plan we do not want to show this to anyone

Reply:
OK, then it sounds good like this.